airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -607,5 +621,16 @@ def execute(self, context, session=None):
                               traceback.format_exc(),
                               extra={'err': ex})
 
-            self.clean_up(job_name)
+                result = subprocess.check_output(args=namespaced_kubectl() +
+                                                 ['describe', 'job', job_name])
+                logging.info("Describing failed job {}".format(result))
+
+                pod_output = self.get_pods(job_name)
+                for pod in pod_output['items']:
+                    pod_name = pod['metadata']['name']
+                    result = subprocess.check_output(
```

Comment:
This should be in a `try/except` block. We don't want to miss clean-up if this fails. Alternatively, we could put the clean-ip call in a `finally` block to avoid any potential issue.